common/jenkins-agents/maven/docker/Dockerfile.centos7

```diff
@@ -23,9 +23,9 @@ ENV BASH_ENV=/usr/local/bin/scl_enable \
     GRADLE_USER_HOME=/home/jenkins/.gradle
 
 # Install Java
-RUN yum install -y java-11-openjdk-devel && \
-    yum clean all -y && \
-    exactVersion=$(ls -lah /usr/lib/jvm | grep "java-11-openjdk-11.*\.x86_64" | awk '{print $NF}' | head -1) && \
+RUN yum install -y epel-release && yum repolist && yum install -y java-latest-openjdk-devel && \
```

Comment:
This install the EPEL repository (Extra Pacakges for Enterprise Linux, see https://www.cyberciti.biz/faq/installing-rhel-epel-repo-on-centos-redhat-7-x/), otherwise the package `java-latest-openjdk-devel` is not found. Opposed to centos 8, the java package name in centos 7 called `latest`.